docs/source/x_ext.rst

```diff
@@ -630,6 +636,47 @@ Whether a load or store is treated as being speculative or not by the |processor
 ignore whatever value it might have communicated via ``commit_kill`` with respect to whether it treats a memory request as speculative or not. A |coprocessor|
 is allowed to signal ``spec`` = 1 without taking the commit transaction into account (so for example even after ``commit_kill`` = 0 has already been signaled).
 
+The ``addr`` signal indicates the (byte) start address of the memory transaction. Transactions on the memory (request/response) interface cannot cross a X_MEM_WIDTH (bus width) boundary.
```

Comment:
I am unsure if this is already stated elsewhere, but in order to avoid confusion it might worthwhile to clarify that although `addr` can use the byte start address of the transaction, the byte lanes of the data signal (and hence also the bits of the `be` signal) are actually aligned to the width of the memory interface `X_MEM_WIDTH`.

Reply:
I will add that remark.